Solutions/Intel471/Playbooks/Intel471-ImportMalwareIntelligenceToSentinel/azuredeploy.json

```diff
@@ -0,0 +1,1074 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "comments": "This playbook ingests malware indicators from Intel 471's Titan API into ThreatIntelligenceIndicator table.",
+        "author": "Marcin Molenda, Intel 471, Inc."
+    },
+    "parameters": {
+        "PlaybookName": {
+            "defaultValue": "intel471-malware-indicators",
+            "type": "String"
+        },
+        "TitanUserName": {
+            "defaultValue": "",
+            "type": "String"
+        },
+        "TitanAPIKey": {
+            "defaultValue": "",
+            "type": "SecureString"
+        },
+        "TargetProduct": {
+            "defaultValue": "Azure Sentinel",
+            "type": "String",
+            "metadata": {
+                "description": "Security product to which the indicators will be applied. Allowed values: Azure Sentinel, Microsoft Defender ATP."
+            }
+        },
+        "Action": {
+            "defaultValue": "unknown",
+            "type": "String",
+            "metadata": {
+                "description": "The action to apply if the indicator is matched from within the targetProduct security tool. Allowed values: unknown, allow, block, alert."
+            }
+        },
+        "LookBackDays": {
+            "defaultValue": 0,
+            "type": "int",
+            "metadata": {
+                "description": "How many days of history should be pulled on the first run. Leave 0 to start from the current time."
+            }
+        }
+    },
+    "variables": {
+        "GraphSecurityConnectionName": "[concat('microsoftgraphsecurity-', parameters('PlaybookName'))]",
+        "AzureBlobConnectionName": "[concat('azureblob-', parameters('PlaybookName'))]",
+        "AzureStorageName": "[concat('i471sa', substring(uniqueString(resourceGroup().id),0,13))]",
+        "AzureKeyVaultName": "[concat('i471kv', substring(uniqueString(resourceGroup().id),0,13))]",
+        "AzureKeyVaultConnectionName": "[concat('keyvault-', parameters('PlaybookName'))]",
+        "ContainerName": "default"
+    },
+    "resources": [
+        {
+            "type": "Microsoft.KeyVault/vaults",
```

Comment:
In template, we don't support creation of key vault, storage account and other resources, kindly try to use the pre-existing storage account and key vault, 

try to convince customer to create key vault and storage account manually by giving the steps on on readme file